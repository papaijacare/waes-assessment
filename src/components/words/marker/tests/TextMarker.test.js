import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import configureStore from 'redux-mock-store'

import { TextMarker } from '../..'

const mockStore = configureStore()
const store = mockStore({
    colors: { 
        filteredColors: [], 
        selectedColor: undefined
    }
})

const instance = () => {
    const props = {
        store: store,
    }
    const wrapper = shallow(<TextMarker { ...props }/>)
    
    return {
        wrapper
    }
}

describe('TextMarker', () => {
    test('renders correctly', () => {
        const { wrapper } = instance()
        const component = wrapper.dive()

        expect(toJson(component)).toMatchSnapshot()
    })
})