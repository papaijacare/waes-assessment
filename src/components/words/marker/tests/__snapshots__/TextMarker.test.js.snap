// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextMarker renders correctly 1`] = `
<Segment>
  <div
    contentEditable={true}
    onClick={[Function]}
    onMouseUp={[Function]}
    suppressContentEditableWarning={true}
  >
    Laborum minim excepteur do irure eu ea occaecat ipsum eiusmod consequat consequat id Lorem. Aliqua fugiat sit deserunt ea elit mollit eu in do ut ad nostrud cupidatat. Culpa deserunt excepteur duis ut non aliqua cupidatat ipsum eu.
  </div>
</Segment>
`;
